packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -276,37 +309,39 @@ class AccountSwitcher extends React.Component {
                                 </div>
                             ) : (
                                 <React.Fragment>
-                                    {this.props.has_mt5_login && (
-                                        <div
-                                            className={classNames('acc-switcher__accounts', {
-                                                'acc-switcher__accounts--has-add-account': this
-                                                    .has_demo_mt5_upgrade_button,
-                                            })}
-                                        >
-                                            {this.sorted_mt5_list
-                                                .filter(account => /^demo/.test(account.group))
-                                                .map(account => (
-                                                    <AccountList
-                                                        key={account.login}
-                                                        account_type={account.group}
-                                                        balance={account.balance}
-                                                        currency={account.currency}
-                                                        currency_icon={`IcMt5-${getMT5AccountDisplay(account.group)}`}
-                                                        has_balance={'balance' in account}
-                                                        is_virtual
-                                                        loginid={account.display_login}
-                                                        onClickAccount={this.redirectToMt5Demo}
-                                                    />
-                                                ))}
+                                    {this.demo_mt5.length ? (
```

Comment:
No need for ternary:
```suggestion
                                    {!!this.demo_mt5.length && (
```